src/NBitcoin/BitcoinCore/CoinsView.cs

```diff
@@ -106,7 +106,7 @@ public CoinsView CreateCached()
             return new CoinsView(new CachedNoSqlRepository(this.Index));
         }
 
-        public void AddTransaction(Transaction tx, int height)
+        public void AddTransaction(Consensus consensus, Transaction tx, int height)
```

Comment:
`Consensus consensus` isnt being used?

Reply:
Ah yeah it was going in the constructor and now its not needed.
hmm... but this class is never really used outside of NBitcoin